data/scenario-03.json

```diff
@@ -57,7 +57,7 @@
                       "type": "paragraph",
                       "blockId": "1bf4d5c7-a386-32c5-4ad5-a037ead08760",
                       "model": {
-                        "text": "Lorem ipsum dolor sit amet, consectetur adipiscing elit ©"
+                        "text": "Lorem ipsum dolor sit amet, consectetur adipiscing elitelit ©"
```

Comment:
Did you mean to make the text change here?

Reply:
you can reject this one. I did in my 'github learning curve' while trying to understand if I can make changes in different files under the same PR :)